docs/CMS-SETUP.md

```
@@ -2,41 +2,64 @@
 
 Sveltia CMS is mounted at `gvns.ca/admin`. Auth is brokered by the `auth.gvns.ca` Worker (GitHub OAuth).
 
-## Media uploads
+## Posts use the bundle layout
 
-All CMS-uploaded media (currently just `heroImage`) lands in `/public/uploads/` and is referenced by absolute URL in frontmatter:
+Each post is its own folder. The post body is `index.md`; uploaded images sit beside it as siblings:
 
-```yaml
-heroImage: /uploads/my-photo.jpg
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)